tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/VariableReturnType.java

```diff
@@ -344,6 +347,24 @@ public static Object getWithUnion(Object x, BTypedesc y) {
         return ErrorCreator.createError(StringUtils.fromString("Error!"), StringUtils.fromString("Union typedesc"));
     }
 
+    public static BMap<BString, Object> getAnnotationValue(BTypedesc y) {
+        return ((BTypeReferenceType) y.getDescribingType()).getAnnotations();
+    }
+
+    public static Object getAnnotationValue2(Object value, BTypedesc y, BString annotationName, int min, int max) {
```

Comment:
Better if the variable name `y` can be renamed to something meaningful such as `typeDesc`.